### DISCUSSION TITLE
Awesome extension, here's my MOD repro

### DISCUSSION THREAD

**Discussion by halr9000** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#54] [Show and tell]
My aim was to reproduce the MOD house/path/robot [motivation ](https://github.com/albarji/mixture-of-diffusers#motivation)example with these prompts:

"A charming house in the countryside, by jakub rozalski, sunset lighting, elegant, highly detailed, smooth, sharp focus, artstation, stunning masterpiece"
"A dirt road in the countryside crossing pastures, by jakub rozalski, sunset lighting, elegant, highly detailed, smooth, sharp focus, artstation, stunning masterpiece"
"An old and rusty giant robot lying on a dirt road, by jakub rozalski, dark sunset lighting, elegant, highly detailed, smooth, sharp focus, artstation, stunning masterpiece"

Using your new region control feature like this:
![image](https://user-images.githubusercontent.com/18482/227206250-daae5edd-6d1b-49e3-bca0-6424282093b7.png)

I was easily able to produce this!
![image](https://user-images.githubusercontent.com/18482/227206322-403321c3-c6e1-4105-b041-2feb14bb425a.png)

**Comment by pkuliyi2015** [comment]
Nice work!

Currently I have add some new tech into the repo and removed all necessary settings, you can have a try on my latest version.
